.gitignore

```
@@ -104,4 +104,6 @@ venv.bak/
 # PyPI configuration file
 .pypirc
 
-bot.toml
\ No newline at end of file
+bot.toml
+data/
+data/club.json
```

```suggestion
data/
```
Si le dossier est gitignore, tout le contenu est gitignore aussi